apps/condo/admin-ui/index.js

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
```

Comment:
do we really need it for js files?

Reply:
git hub warnings :-)
![image](https://user-images.githubusercontent.com/1640424/168091747-c06b1059-5b43-47c6-9b35-58ecf9d9f09b.png)

Reply:
:)) GitHub think that we use TS